routes/index.js

```diff
@@ -225,80 +237,76 @@ module.exports = function (passport) {
       endDate = new Date(endRange);
       endDate.setDate(endDate.getDate() + 1);
     }
-    
+
     Promise.all([
       dbAPI.getUsers(startDate, endDate),
       dbAPI.getPrivilegedStationUsers(req.body.stationSelector),
       dbAPI.getStations()
-    ])
-
-      .then(
-        ([usersByDate, privUsers, allStations]) => {
-          if (!usersByDate) {
-            const err = "Interal error: Unable to get users."
-            req.flash('error', err);
-            res.status(500).send(err + "via the /userManagement (post) route.");
-            res.redirect('/userManagement');
-          }
+    ]).then(
+      ([usersByDate, privUsers, allStations]) => {
+        if (!usersByDate) {
+          const err = "Interal error: Unable to get users."
```

Comment:
I just wanted to make sure I'm remembering correctly: we made a conscious decision not to try and log internal errors like this, right? Like - its okay to display a flash message like it is... but if the DB is down and we try to log, it could spin into a perpetual cycle? I ask because I think I did that somewhere else a while back and if so, I need to find it.

Reply:
Yes I believe this is the case.